src/akamai/js/akamaiHD_flash.js

```diff
@@ -483,8 +483,8 @@ require("../../../html5-common/js/utils/constants.js");
       self.controller.notify(self.controller.EVENTS.SEEKED);
     };
 
-    var raiseBufferingEvent = function() {
-      self.controller.notify(self.controller.EVENTS.BUFFERING);
+    var raiseBufferingEvent = function(event) {
+      self.controller.notify(self.controller.EVENTS.BUFFERING, { url: event.eventObject.url });
```

Comment:
OK, but let's do it right away please so that it doesn't slip through the cracks.  And, actually - this is not for current fix, but...  There are no unit tests for these plugins, and many problems, including this one would have been discovered had we had unit tests.